Makefile

```diff
@@ -406,3 +406,14 @@ install-rpm: install-common
 
 	# devel
 	$(INSTALL_DATA) -D key/aziot-keys/aziot-keys.h $(DESTDIR)$(includedir)/aziot-identity-service/aziot-keys.h
+
+install-rpm-mariner: install-common
```

Comment:
The only difference from `install-rpm` seems to be not installing the unit preset, so the specfile should just use `install-rpm` and then delete the `install`ed preset.

Reply:
The main issue with that is that 00-aziot.preset is not a file that is copied over for Mariner at install time. this will result in the make command failing. If you think it is still worth it we can add 00-aziot.preset to be copied over and use the same install-rpm make call.

Reply:
Removed install-rpm-mariner. there is now a flag that will determine if the preset gets installed or not

Reply:
You don't need a flag. Just `rm` after you call `make install` in the new specfile.

Reply:
The problem is getting the preset file to the correct location because it will fail on finding it without any change. We would have to copy it to wherever ../../Sources is on the mariner tooIkit before calling make, personally feel like that is not an ideal solution but will implement that as well.

Reply:
Right, you would have to include the preset in the source too. It's true that it's odd to do that. I'm okay with the flag.